src/accessibility/describe.js

```diff
@@ -178,44 +178,45 @@ p5.prototype.describeElement = function(name, text, display) {
   name = name.replace(/[^a-zA-Z0-9 ]/g, '');
   //store element description
   let inner = `<th scope="row">${elementName}</th><td>${text}</td>`;
-
-  //if it is the first time describeElement() is called
-  if (
-    !dummy.fallbackElements[cnvId + name] ||
-    !dummy.labelElements[cnvId + name]
-  ) {
-    //store copy of body dom in dummy
-    _populateDummyDOM(cnvId);
+  //if there is no dummyDOM
+  if (!this.dummyDOM) {
+    this.dummyDOM = document.getElementsByTagName('body')[0];
```

Comment:
does this mean that we always expect the element to be the first item appended to the body? what if the user has specified a different/custom place for the canvas?